### DISCUSSION TITLE
WhatsApp uygulaması ricası

### DISCUSSION THREAD

**Discussion by nevisende** [acikkaynak/deprem-yardim-projesi#41] [Q&A]
Arkadaşlar sizden ricam deprem bölgesinden biri olarak,
 lütfen wp üzerinden bir uygulama yapın
 Eczanem gibi insanlar konumumu atınca ve ihtiyacını ne olduğunu yazınca çevresinde  bulabileceği konumları görsün 
 >başka numaradan da insanlar konumunu atsın hangida yardım yapacaksa çevresindeki nerde nekadar kişi istediğini gösteren bir map veya map linki cevabı alsın.
Herkese teşekkürler depremzede kardeşlerim adına Allahın izniyle daha fazlasını da kurtaracağız sizden ricam yukarıdaki önerimi soğukta ne yapacağını bilemeyen aracı bile olmayan ama kapıda yığınla yardım bekleyen insanlar için değerlendirmenizdir, insanlar hangi siteden yardım buluruzdan  çok kimi ararız kime yazarız diyorlar buranın insanların çoğunluğu internette farklı sitelerdr gezmez afadın sitesine bile bakmayı düşünmüyorlar diyeyimki anlayın çaresiz olsada böyle şeylerle uğraşmadıklarını

**Comment by sercangoger** [comment]
proje yöneticileri buraya baksın acil

**Comment by mtkucuk** [comment]
proje yöneticileri buraya bakarmı acil bir şekilde.
teknik süreçte yardıma açığım.

**Comment by selamikarakas** [comment]
Geliştirilmiş bir whatsapp botu var yayalım lütfen. https://wa.me/message/FZ5OLBPJLZMIP1

**Reply by nevisende** [reply]
Şaka mı yapıyorsunuz bu botla napalım söyler misiniz burda insanlar yemek kıyafet çadır bot temizlik malzemeleri kömür arıyor bu botta ne var Allah için ki yayayım birine bu numarayı önersem 2 dk sonra yüzüme tükürür alaymı ediyorsun diye onu da geçtim yardım etmek için yollara çıkan onca insan var onların kullanacağı bot nerde peki

**Reply by selamikarakas** [reply]
Haklısınız, bu bot üzerinden geliştirmeler dediğiniz gibi devam ederse çok güzel olur. Çok değerli tavsiyeleriniz var. Umarım botu geliştiren ekip görür.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebinizi bu repoların [1](https://github.com/acikkaynak/deprem-whatsapp-bot) ve [2](https://github.com/acikkaynak/deprem-yardim-whatsapp) issues veya pr alanına taşıyabilirsiniz.

**Reply by heyaygun** [reply]
Arkadaslar Mert isimli 16 yasinda bir kardesimiz gelistirmis dun konustum. Bunu hizlandirabiliriz. Whatsapp tarafinda destek gerekirse bana ulasabilirsiniz. ![image](https://user-images.githubusercontent.com/1314515/218270543-a32f3ada-0441-4ef4-b8d5-9921a71e0143.png)

**Comment by MertBalkan** [comment]
++
